src/pipecat/frames/frames.py

```diff
@@ -615,6 +633,13 @@ def __str__(self):
         return f"{self.name}(pts: {pts}, text: [{self.text}], size: {self.size}, format: {self.format})"
 
 
+@dataclass
+class InputDTMFFrame(Frame):
```

Comment:
This should be a `DataFrame`. Inheriting from `Frame` directly is only done by `DataFrame`, `ControlFrame` and `SystemFrame`.

```
class InputDTMFFrame(DataFrame):
```

Reply:
Also, I was thinking if we should move this to the twilio file and define the frame there. But this seems generic enough that could live here.